src/murmur/Messages.cpp

```diff
@@ -290,6 +345,12 @@ void Server::msgAuthenticate(ServerUser *uSource, MumbleProto::Authenticate &msg
 
 	// Setup UDP encryption
 	{
+		if (uSource->vptVoiceProtocolType == VoiceProtocolType::UNDEFINED) {
+			// If no Capabilities message has been received
+			uSource->vptVoiceProtocolType = VoiceProtocolType::UDP_OCB2;
+			uSource->initializeCipher();
+		}
```

Comment:
If the client version is high enough to know about the capabilities message, the server should disconnect the client instead of falling back to OCB2.

Depending on when we'll merge this PR the client version would be 1.4.0 or 1.5.0

Reply:
Sure. :)